packages/repository/src/common-types.ts

```diff
@@ -100,7 +100,7 @@ export interface Count {
  */
 export const CountSchema = {
   type: 'object',
-  title: 'loopback.count',
+  title: 'Loopback.Count',
```

Comment:
The namespace is preferred to be lower case. In this case, I would like to see `loopback.Count`.